conformance/packages/dns-test/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+edition = "2021"
+license = "MIT OR Apache-2.0"
+name = "dns-test"
+publish = false
+version = "0.1.0"
+
+[dependencies]
+lazy_static = "1.4.0"
```

Comment:
given that we are using latest stable (1.78) instead of this project's MSRV (1.67) to run the conformance test suite, we can indeed use `std::OnceLock` (1.70) instead of `lazy_static!`. that'd be a good refactor to do after this PR is merged

Reply:
I'm not sure we "need" to change this, but, 1.70 was released a year ago at this point, I don't have any problem with us upgrading that, I think we really only have a mildly enforced goal of about 6 months or -3 releases.